packages/format-library/src/link/index.native.js

```diff
@@ -108,10 +111,25 @@ export const link = {
 			speak( __( 'Link removed.' ), 'assertive' );
 		}
 
+		async getURLFromClipboard() {
+			const clipboardText = await Clipboard.getString();
+			if ( ! clipboardText ) {
+				return '';
+			}
+			// Check if pasted text is URL
+			if ( ! isURL( clipboardText ) ) {
+				return '';
```

Comment:
```suggestion
				return;
```

Nothing is reading a return value from this function